src/analysis/retail/evoker/shared/modules/talents/hero/flameshaper/FanTheFlames.tsx

```diff
@@ -0,0 +1,161 @@
+import { FAN_THE_FLAMES_INCREASE } from 'analysis/retail/evoker/shared/constants';
+import SPELLS from 'common/SPELLS';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Combatant from 'parser/core/Combatant';
+import Enemy from 'parser/core/Enemy';
+import { calculateEffectiveDamage, calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import Events, {
+  ApplyBuffEvent,
+  ApplyDebuffEvent,
+  CastEvent,
+  DamageEvent,
+  HealEvent,
+  RemoveBuffEvent,
+  RemoveDebuffEvent,
+} from 'parser/core/Events';
+import { Options } from 'parser/core/EventSubscriber';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+import Combatants from 'parser/shared/modules/Combatants';
+import Enemies from 'parser/shared/modules/Enemies';
+import SpellLink from 'interface/SpellLink';
+import { formatNumber } from 'common/format';
+import SPECS from 'game/SPECS';
+
+class FanTheFlames extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    enemies: Enemies,
+  };
+
+  totalDamage: number = 0;
+  totalHealing: number = 0;
+  numExtended: number[] = [];
```

Comment:
nit - the hots aren't extended, can we call this `numAmplified` or something similar